src/app/component/create-loan/create-loan.tsx

```diff
@@ -1,66 +1,224 @@
-                                className='bg-transparent
-                                focus:border-transparent outline-none 
-                                cursor-pointer'>
-                                {loanPeriodValue.map((period, idx) => (
-                                    <option key={idx} value={period.value}>{period.name}</option>
-                                ))}
-                            </select>
-                        </div>
-                    </div>
+  const handleRequestAmountOptionChange = (
+    e: React.ChangeEvent<HTMLSelectElement>
+  ) => {
+    const selectedValue = e.target.value;
+    const selectedOption = requestAmountOptions.find(
+      (option) => option.value === selectedValue
+    );
+    setSelectedRequestAmountOption(selectedOption || null);
+  };
+
+  const handleCollateralOptionChange = (
+    e: React.ChangeEvent<HTMLSelectElement>
+  ) => {
+    const selectedValue = e.target.value;
+    const selectedOption = collateralAmountOptions.find(
+      (option) => option.value === selectedValue
+    );
+    setSelectedCollateralAmountOption(selectedOption || null);
+  };
+
+  return (
+    <section className="">
+      <div className="w-full h-full mx-auto flex flex-col gap-5">
+        <h3 className="text-textBlack text-lg font-semibold font-raleway leading-[23.4px] mb-2">
+          Create Loan position
+        </h3>
+        <div className="flex flex-col gap-4">
+          <div className="request-amount flex flex-col gap-2 relative w-full">
+            <h4 className="text-textBlack text-[16px] tracking-[0.16px] leading-[20.08px] font-inter font-medium my-1">
+              Request Amount
+            </h4>
+            <div className="bg-gray-100 px-2 py-4 flex rounded-2xl relative">
+              <input
+                type="text"
+                placeholder="3,000.04"
+                className="text-textBlack bg-transparent border border-transparent w-[55%] focus:outline-none"
+                value={requestAmount}
+                onChange={(e) => setRequestAmount(e.target.value)}
+              />
+              <div className="w-[45%] flex gap-4 justify-end self-center relative">
+                <span className="h-full text-[#C3C8CA]">{"|"}</span>
+                <div className="flex justify-center items-center relative">
+                  <AiOutlineDollarCircle className="h-7 w-7 inline text-appColor1" />
+                  <select
+                    name="requestToken"
+                    id="requestToken"
+                    className="bg-transparent cursor-pointer focus:border-transparent outline-none"
+                    onChange={handleRequestAmountOptionChange}
+                  >
+                    {requestAmountOptions.map((opt, idx) => (
+                      <option key={idx} value={opt.value}>
+                        {opt.name}
+                      </option>
+                    ))}
+                  </select>
```

Comment:
We can not have select spread on all the code, we need a common global input

Reply:
done